public/js/monitor.js

```diff
@@ -458,3 +463,9 @@ if (document.querySelectorAll("button")) {
     button.addEventListener("click", (e) => doButtonRouting(e));
   });
 }
+
+if (document.querySelector(".breach-image")) {
```

Comment:
Not sure what this `if` is doing. IIUC, the `document.querySelectorAll()` would return an empty array if the selector wasn't found, so the `.forEach()` would be a noop. Not sure we need the extra check here.